spec/components/email/receiver_spec.rb

```diff
@@ -341,13 +341,21 @@ def create_post_reply_key(value)
     end
 
     it "works" do
```

Comment:
Sorry I know it's not on you, but what does "works" mean? Can we elaborate in the description?

Reply:
It's done 👍